DeepSpeech/import_m-ailabs.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -xe
+
+pushd $HOME/ds/
+	if [ "${ENGLISH_COMPATIBLE}" = "1" ]; then
+		IMPORT_AS_ENGLISH="--normalize"
+	fi;
+
+	if [ ! -f "/mnt/extracted/data/M-AILABS/it_IT/it_IT_train.csv" ]; then
+		# added --skiplist nothing to avoid issues with the importer TODO: find a proper solution
+		python bin/import_m-ailabs.py ${IMPORT_AS_ENGLISH} \
+		    --skiplist nothing\
```

Comment:
@lissyx I had to do this silly stuff in order to not receive errors from import_m-ailabs.py. Any idea on how to fix this?

Reply:
I could help you if you told me what error you got...

Reply:
@mone27 Looking at the code of `bin/import_m-ailabs.py` I suspect you got an issue at `CLI_ARGS.skiplist.split(',')` ?

Could you file a bug on DeepSpeech repo and send a PR ? I suspect it should be enough to set a default value to the string in `parser.add_argument('--skiplist', type=str, help='Directories / books to skip, comma separated')`